index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h2>Hello world, we are software developers!</h2>
+    <div id="containerpage">
+      <header id="header">
+        <div class="hlogo">
+          <img
+            id="header-img"
+            src="images/triallogo.png"
+            ;
+            alt="original trombones logo"
+          />
+        </div>
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#Features">Features</a></li>
+            <li><a class="nav-link" href="#How It Works">How It Works</a></li>
+            <li><a class="nav-link" href="">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div style="margin-top: 100px; margin-left: 50px">
+        <iframe
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/y8Yv4pnO7qc"
+          title="Roman Carnival Overture Op. 9 for Five Trombones"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </div>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
```

Comment:
If you forget the method name of the form  I would like to remind you.
```suggestion
      <form id="form" action="https://www.freecodecamp.com/email-submit">
```

Reply:
- [ ] Please be clear in your comments, you pointed out that the person being reviewed forgot the method attribute of the form, but you're not requesting any change with that information! 🤓 🔧 

>This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
>
>This is a vague comment: The README is not descriptive, why don't you make it better?

Reply:
For your form elements, You have specified action attributes explicitly that are good things. But the Method attribute 
has not been specified explicitly. the browser uses the default attribute value means "GET".
The method attribute specifies how to send form data (the form data is sent to the page specified in the action attribute).
The form data can be sent as URL variables (with method="get") or as HTTP post transaction (with method="post")

 # Notes on GET:
Appends form data into the URL in name/value pairs
- The length of a URL is limited (about 3000 characters)
- Never use GET to send sensitive data! (will be visible in the URL)
- Useful for form submissions where a user wants to bookmark the result
- GET is better for non-secure data, like query strings in Google
If you do not agree with above mention please can you use the POST method attribute?
 https://www.w3schools.com/tags/att_form_method.asp read more about post